.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,82 @@
+{
+	"name": "Home Ops",
+	"image": "mcr.microsoft.com/devcontainers/base:debian",
+	"containerEnv": {
+		"KUBECONFIG": "${containerWorkspaceFolder}/kubeconfig",
+		"SOPS_AGE_KEY_FILE": "${containerWorkspaceFolder}/.age.key"
+	},
+	"postCreateCommand": {
+		// Install additional python dependencies for ansible
+		"deps": "task deps"
+	},
+	"features": {
+		// Upgrade OS packages
+		"ghcr.io/devcontainers/features/common-utils:2": {
+			"version": "latest",
+			"installZsh": false,
+			"installOhMyZsh": false
+		},
+		// Install go-task
+		"ghcr.io/devcontainers-contrib/features/go-task:1": {
+			"version": "3.31.0"
+		},
```

Comment:
```suggestion
		"ghcr.io/devcontainers-contrib/features/go-task:1": {},
```
Do you know if this would always pull the latest version?

Reply:
If a version is provided it will grab that version. If the version provided is latest it will always pull latest on rebuild of the devcontainer. If nothing is provided it's whatever the default is. The convention is default version to latest.

Reply:
Nice, I would go with this for all the deps then since that is what brew does anyways.

Reply:
Pinning versions is a habit from asdf but I am mostly fine with latest on everything. Though @onedr0p are you concerned about any kubectl latest breaking for someone with an older cluster?

Reply:
I'm not concerned, if someone runs into an issue hopefully they are able to figure out why and pin it themselves. I try to maintain support for the latest version of all tools being used here.